#react-tinymce

#### Category
- [ ] Question
- [x] Bug
- [ ] Enhancement

#### Expected or Desired Behavior
The webpart react-tinymce have some minimal issues at the beginning when gulp serve is executed.
Some lines are missing and of this the application need to be modified first before running gulp serve.
The application is an texteditor and this means that height and position is needed for the webpart inside the SCCS module file.

#### Observed Behavior
An error occures when running at the first time after executing gulp serve:
[10:18:43] Error - typescript - src\webparts\reactTinyMce\ReactTinyMceWebPart.ts(15,25): error TS2307: Cannot find module './components/ReactTinyMce.module.scss'.
[10:18:43] Error - typescript - src\webparts\reactTinyMce\components\ReactTinyMce.tsx(16,19): error TS2307: Cannot find module './ReactTinyMce.module.scss'.

This happenes of the reason for: 
.reactTinyMce {
// An emplty SCSS class that affects the application to run at the first time. This is a texteditor.
}

#### Steps to Reproduce
The application should have some parameters inside the module:
.\react-tinymce\src\webparts\reactTinyMce\components\ReactTinyMce.module.scss

Default values that may affect the first time run by gulp serve:

.reactTinyMce {

}

#### Submission Guidelines

First time runnig the application will generate an error because of some error inside the component module. I found that under the ReactTinyMce.module.scss file that has empty parameters that should not have. 

What I did was to put this inside the file and the application worked:

.reactTinyMce {
    max-width: 100%;
    margin: 0 auto;
}
This reason is of the fact that the texteditor must have some margin and height before the project runs. Otherwise there will be no reason by the workbench to create the webpart without. Text editors must have edges and margins from the start.

Keep in mind that you can still have the application running and just enter the parameters inside the scope. Just save and wait until the page is reupdated.

Thank you! Sharing is caring.

--- a/samples/react-tinymce/src/webparts/reactTinyMce/components/ReactTinyMce.module.scss
+++ b/samples/react-tinymce/src/webparts/reactTinyMce/components/ReactTinyMce.module.scss
@@ -1,5 +1,5 @@
 @import '~@microsoft/sp-office-ui-fabric-core/dist/sass/SPFabricCore.scss';
 
 .reactTinyMce {
-  
+  background: inherit;
 }

